src/DynamoCore/Models/RecordableCommands.cs

```diff
@@ -1583,12 +1583,26 @@ public ModelEventCommand(string modelGuid, string eventName, int value = 1)
                 Value = value;
             }
 
+            [JsonConstructor]
+            public ModelEventCommand(string modelGuid, string eventName)
```

Comment:
At least in C++, it's not an error until you try to call it.  I'm not sure about the C# rules here.